modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/model-source.mustache

```diff
@@ -245,26 +245,29 @@ void {{classname}}::fromJson(web::json::value& val)
         {{#isString}}
         {{setter}}(ModelBase::stringFromJson(val[utility::conversions::to_string_t("{{baseName}}")]));
         {{/isString}}
+        {{#isByteArray}}{{setter}}(ModelBase::stringFromJson(val[utility::conversions::to_string_t("{{baseName}}")]));{{/isByteArray}}
```

Comment:
stringFromJson does not exist, according to your header

Reply:
Will check.

Reply:
@stkrwork 
I checked it and it is there actually I generated and tested this code 
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/modelbase-header.mustache#L52

Reply:
I tested using the spec from #407 for `cpp-rest-sdk`

Reply:
I checked this, and this is the modelbase header from your pr:
https://github.com/etherealjoy/openapi-generator/blob/4098c7ae299bb8e0e0b31a6beb37fbcb08700a81/modules/openapi-generator/src/main/resources/cpp-pistache-server/modelbase-header.mustache

i cant seem to find the function there.

Reply:
@stkrwork 
This change is in `cpp-rest-sdk-client`
I simply combined both the fixes in one.

Reply:
oh nvm, i didn't see the cpp-rest-sdk-client in the path, my bad

Reply:
Next time, please seperate it into two pull requests, or change the title, because i was not expecting cpprestsdk code in the pr

Reply:
Yes, sorry for the confusion.